lms/djangoapps/course_api/blocks/transformers/milestones.py

```diff
@@ -66,8 +66,6 @@ def user_gated_from_block(block_key):
 
             if usage_info.has_staff_access:
                 return False
-            elif self.has_pending_milestones_for_user(block_key, usage_info):
-                return True
```

Comment:
hmm.. this API is also used by Mobile apps.  Currently, the mobile apps don't present gated content in the course outline.  Do we want this behavior to change for mobile apps as well?

If we don't, then we'll need to add a condition for whether or not to exclude gated content - similarly to how this was done for special exams via the `include_special_exams` boolean parameter.

Reply:
@nasthagiri @marcotuts 
I would think we'd want the same behavior change for mobile (i.e. show gated content in outline) so we are consistent. Any concerns here?

Reply:
in that case, we will need to make sure 
(1) the mobile app presents the right thing (?) once the gated subsection shows in the outline and learners click on it
(2) the change in API works for both old and new mobile apps, without causing much confusion or errors